app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,82 @@
+<section class="signPage">
+<div class="header">
+  <%= link_to "<i class='fas fa-arrow-left'></i>".html_safe, :back, class: "btn" %>
+  <h2>Sign Up</h2>
+  <%= link_to "Log In", categories_path, class: "btn" %>
+</div>
+
+<div class="form">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field name">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="bottom">
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
+</div>
+</section>
+
+<style>
+  .header {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 60px;
+    padding: 8%;
+    background: #3778c2;
+    color: aliceblue;
+    gap: 3rem;
+
+  }
+
+  .header a{
+    color: aliceblue;
+    text-decoration: none;
+    font-weight: bold;
+  }
+
+  .form {
+    margin-top: 5%;
+  }
+
+  #user_name {
+    height: 45px;
+    width: 100%;
+    padding: 4%;
+  }
```

Comment:
- [ ] I would suggest that you remove the styling properties applied on the user_name id input field. I would encourage you to put more effort into improving the user interface of this application because this is a significant portfolio project.

- [ ] Although this project is a mobile application, you should also optimize it for viewing on a desktop device.

Reply:
the issue is on render, the styling isn't fully loaded, because I tried to deploy the app dynamically but it was asking for a credit card even though it's free to use, so I was obliged to do it manually which caused the problem in the styling,
if you tried and run it at your localhost and adjust the mobile dimension everything is gonna work alright.